src/main/java/no/finn/unleash/Unleash.java

```diff
@@ -15,6 +16,18 @@ default boolean isEnabled(String toggleName, UnleashContext context, boolean def
         return isEnabled(toggleName, defaultSetting);
     }
 
+    default boolean isEnabled(final String toggleName, final BiFunction<String, UnleashContext, Boolean> fallbackAction) {
+        return isEnabled(toggleName, false, fallbackAction);
+    }
+
+    default boolean isEnabled(String toggleName, UnleashContext context, boolean defaultSetting, BiFunction<String, UnleashContext, Boolean> fallbackAction) {
```

Comment:
This one is not needed. It does not make sense to allow both defaultSetting and a fallbackAction.